packages/config/src/options.ts

```diff
@@ -664,11 +664,6 @@ export const breakingRootOptions: Array<BreakingOption> = [
     errorKey: 'CONFIG_FILE_INVALID_ROOT_CONFIG',
     isWarning: false,
     testingTypes: ['e2e'],
-  }, {
```

Comment:
I believe the typescript definitions are incorrect and need to moved from E2E [here](https://github.com/scottohara/cypress/blob/issue-25636-run-all-specs-for-component-testing/cli/types/cypress.d.ts#L3276) and over to all resolved config options [here](https://github.com/scottohara/cypress/blob/issue-25636-run-all-specs-for-component-testing/cli/types/cypress.d.ts#L3185)
<img width="570" height="245" alt="Screenshot 2025-12-11 at 12 02 57 PM" src="https://github.com/user-attachments/assets/d1bae605-bcce-42a8-9a74-1c1bfeeec5aa" />